### ISSUE TITLE
[Metaeffect Suggestion] Effect Buffer

### ISSUE THREAD

**Issue by SeanMirrsen** [gta-chaos-mod/ChaosModV#516]
Alternate names: Calm before the Storm, Hold and Release

Duration: Long

The metaeffect will prevent regularly voted effects from activating, instead putting them into a 'buffer'. Normally 2-3 effects will have been voted on until the metaeffect expires. When the metaeffect expires, all buffered effects are activated at once.

**Comment by TheRealAyCe** [comment]
I'd suggest making voting a bit quicker, otherwise there may be a drought of action for several minutes. Also, can chat see which effects are buffered? Maybe this can replace "no chaos"? Maybe other ideas for what happens during the time?

**Comment by TheRealAyCe** [comment]
Showing the effects to everyone might be even better than hiding it, yes. But it needs to be made very clear why the effects are not active right away, so probably gray or translucent or something.

**Comment by SeanMirrsen** [comment]
Yeah, greyed out and/or blinking would be my suggestion. Combined with the effect name it should be reasonably evident what is going on.
